benchmarks/Bio/fceri_gamma2.jmd

```diff
@@ -21,15 +21,32 @@ using DiffEqBase, OrdinaryDiffEq, Catalyst, ReactionNetworkImporters,
 
 gr()
 const to = TimerOutput()
-tf       = 0.5
+tf       = 150.0

```

Comment:
why?

Reply:
In the Zulip discussion we said we'd run fceri_gamma2 to steady state at 150?

Reply:
From the plots, it looks like it starts at steady state?

Reply:
There was a problem with the old plot where it showed all states, not just the observables. I have changed it so that the plots show the observables instead. When the observables only are shown one can see that it continues until and  SS at around t=150.

Reply:
Makes more sense